students/navdeep/session06/test_args_kwargs_lab.py

```diff
@@ -0,0 +1,32 @@
+from args_kwargs_lab import colors, colors2
+
+def test_1():
+	assert colors('red', 'blue', 'yellow', 'chartreuse') == ('red','blue', 'yellow', 'chartreuse')
+
+def test_2():
+	assert colors(link_color = 'red', back_color = 'blue') == (None, 'blue', 'red', None)
+
+def test_3():
+	assert colors('purple', link_color = 'red', back_color = 'blue') == ('purple', 'blue','red', None)
+
+def test_4():
+	regular = ('red', 'blue')
+	links = {'link_color': 'chartreuse'}
+	assert colors(*regular, **links) == ('red', 'blue', 'chartreuse', None)
+
```

Comment:
How would this test work if someone entered several more arguments for *regular and **links? Can you test for membership instead?